jest.config.js

```diff
@@ -1,9 +1,32 @@
-module.exports = {
+// @ts-check
+/** @type {import('@jest/types').Config.InitialOptions} */
+const comonOptions = {
```

Comment:
I tried specifying these options in the root object (so each project would only have `displayName` and `testMatch`), but that didn't seem to work. So I'm sharing them with object spread instead.

Reply:
This is probably the best we can do!